plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/service/ProviderServiceService.java

```diff
@@ -56,16 +56,6 @@ private static void initProviderService() {
     RecentFileProvider recentProvider = new RecentFileProvider();
     VFSFileProvider vfsProvider = new VFSFileProvider();
 
-    ConnectionManager.getInstance().setMetastoreSupplier( () -> {
-
-      try {
-        return MetaStoreConst.openLocalPentahoMetaStore();
-      } catch ( Exception e ) {
-        // Error getting metastore
-        throw new RuntimeException( e );
-      }
-    } );
-
```

Comment:
Something like this may still be needed, maybe for pan or kitchen? I'm not actually sure. We probably need this in place using the MetastoreLocator, in the end.

Reply:
Agreed. Will do more testing to see if it need the supplier in this case